pootle/apps/pootle_project/models.py

```diff
@@ -339,13 +337,16 @@ def resources(self):
         '''
         cursor = connection.cursor()
         cursor.execute(sql_query, [resources_path, resources_path])
-
         results = cursor.fetchall()
 
         # Flatten tuple and sort in a list
-        resources = list(reduce(lambda x,y: x+y, results))
-        resources.sort(key=get_path_sortkey)
+        resources = set()
+        for result in results:
+            s = result[0]
+            s = s[s.find("/", s.find("/", 1) + 1) + 1:]
+            resources.add(s)
 
+        resources = list(resources).sort(key=get_path_sortkey)
```

Comment:
With this code you never get any results: `.sort()` *has to* go in a separate line because it works in place.

```python
resources = list(resources)
resources.sort(key=get_path_sortkey)
```